src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -45,15 +47,16 @@
 			public static final int ballioElevatorAndIntakeRoller = 3;
 			public static final int climbMotorA = 4;
 			public static final int climbMotorB = 5;
-			public static final int flywheelLeftMotor = 6; // WIP
-			public static final int flywheelRightMotor = 7; // WIP
+			public static final int flywheelMotorA = 6; // WIP
```

Comment:
Why did you change them from flywheel left and right? Wouldn't left and right make more sense, and be easier for debugging when we're having a problem with one motor vs the other? Just a thought.